.github/workflows/rpk-build.yml

```diff
@@ -20,7 +20,7 @@ on:
       - 'v*'
     paths:
       - .github/workflows/rpk-build.yml
-      - 'src/go/rpk/**'
+      - 'src/go/rpk
```

Comment:
this will undo the fix to the pattern for the paths option. It'll be better to leave this change out and only leave the removal of the `helm-chart-release.yml` workflow file